### ISSUE TITLE
data type

### ISSUE THREAD

**Issue by TWTIANTIAN** [pawelsalawa/sqlitestudio#4597]
### Details
Here you describe your problem details, or proposal for enhancement.

### Steps to reproduce
This applies only to bug reports. If this is a feature request, delete this section.

### Operating system
This applies only to bug reports. If this is a feature request, delete this section.

### SQLiteStudio version
This applies only to bug reports. If this is a feature request, delete this section.

**Comment by pawelsalawa** [comment]
Could you describe any details?

**Comment by MooWeii** [comment]
Maybe he wants to say there are fewer data types. It would be handy to provide options for the user to choose which to display in the data type drop-down menu.

**Comment by lallulli** [comment]
I agree: I would add back the standard SQL data types, such as char and varchar, unless the __strict__ option in selected. I use SQLiteStudio for teaching, and I would prefer to show the standard SQL domains.

Furthermore, today I had a bug during my class: I manually entered `char` with a size of `16`, but the size disappeared from the generated SQL code, i.e. it generated `char` instead of `char(16)`. I am trying to reproduce the bug, but now it apparently works.
